### PR TITLE
add `ceph mgr dump` command

### PR DESCRIPTION
**Pull request by trociny**
Signed-off-by: Mykola Golub <mgolub@suse.com>

### CODE REVIEW

**Review by jecluis — COMMENTED** [review]
Wouldn't this be more helpful if it was dumped as json? At least then we could easily, potentially, script obtaining information out of the mgr dump.

**Comment by trociny** [comment]
It already dumps json (ignoring format options)

File: ses [hunk 1]
```diff
@@ -59,6 +59,7 @@ if [ -x /usr/bin/ceph ]; then
     plugin_command "ceph ${ID} --connect-timeout=$CT -s" > $LOG/ceph/ceph-status 2>&1
     plugin_command "ceph ${ID} --connect-timeout=$CT health detail" > $LOG/ceph/ceph-health-detail 2>&1
     plugin_command "ceph ${ID} --connect-timeout=$CT mon dump" > $LOG/ceph/ceph-mon-dump 2>&1
+    plugin_command "ceph ${ID} --connect-timeout=$CT mgr dump" > $LOG/ceph/ceph-mgr-dump 2>&1
```
Comment (sebastian-philipp) [inline ses:62]: Could you quickly verify that the password to the admin account to the dashboard is not written here?

Comment (trociny) [reply ses:62]: I don't see the password in `ceph mgr dump` output (if this is what you asking about).

**Comment by denisok** [comment]
this work for what versions? I am thinking where I should submit new package.

**Comment by trociny** [comment]
It is available since luminous (SES5).

**Comment by denisok** [comment]
@jecluis @sebastian-philipp  please review, so I could land it.

**Review by sebastian-philipp — APPROVED** [review]

**Pull request merged**